src/Stratis.Bitcoin.Tests/Consensus/ConsensusManagerTests.cs

```diff
@@ -348,8 +348,10 @@ public void BlockDownloaded_NullBlock_CallsCallbacks()
             builder.TestConsensusManager.SetupCallbackByBlocksRequestedHash(additionalHeaders.Previous.HashBlock, callback3);
 
             // call for both blocks.
-            builder.blockPullerBlockDownloadCallback(additionalHeaders.Previous.HashBlock, null, peer.Object.Connection.Id);
-            builder.blockPullerBlockDownloadCallback(additionalHeaders.HashBlock, null, peer.Object.Connection.Id);
+            var block = new Block();
```

Comment:
`var block = builder.Network.CreateBlock();` ?

Reply:
Doesn't really matter here, all we need it for is to use  block.SerializedSize